project1/backend/models/resolutionModel.js

```diff
@@ -0,0 +1,86 @@
+const AppError = require('../utils/AppError');
+const formatName = require('../utils/formatName.js');
+const Ajv = require('ajv');
+
+const ajv = new Ajv();
+
+class ResolutionModel {
+  constructor() {
+    this.patients = [];
+    this.patientSchema = {
+      type: 'object',
+      properties: {
+        name: { type: 'string', minLength: 2, maxLength: 20 },
+        resolution: { type: 'string', maxLength: 400 },
+        expiry: { type: 'integer' },
+      },
+      required: ['name', 'resolution', 'expiry'],
+      additionalProperties: false,
+    };
+  }
+
+  create(body) {
+    this.filterTTL();
+
+    // <--- MOVE TO A SEPARATE DTO SECTION ?
+
+    // validation start
+
+    const validateEnqueueSchema = ajv.compile(this.patientSchema);
+    const valid = validateEnqueueSchema(body);
+
+    if (!valid) throw new AppError('Input data is bad.', 404);
+
+    // validation end
+
+    // serialization start
+
+    let { name, expiry, resolution } = body;
+    name = formatName.convertToBabelCase(name);
```

Comment:
For what this convertations?

Reply:
At this moment, the name serves as an ID in req.params that we use when searching for the data in GET/DELETE requests, so the request looks like "GET api/v1/resolutionts/aleksei-leonenko".

Reply:
can we use just lowerCase?